proposals/market_perp_btcusd.go

```diff
@@ -41,12 +41,12 @@ func NewBTCUSDPerpetualMarketProposal(
 				NewMarket: &vega.NewMarket{
 					Changes: &vega.NewMarketConfiguration{
 						DecimalPlaces:           decimalPlaces,
-						PositionDecimalPlaces:   3,
-						LinearSlippageFactor:    "0.1",
-						QuadraticSlippageFactor: "0.1",
+						PositionDecimalPlaces:   4,
+						LinearSlippageFactor:    "0.001",
+						QuadraticSlippageFactor: "0.0",
 						Instrument: &vega.InstrumentConfiguration{
 							Name: name,
-							Code: "BTCUSD.MF21",
+							Code: "BTCUSD.MF21", //TODO (WG): What's the meaning of MF21??
```

Comment:
I guess we could change this to BTCUSD.PERP? Or whatever

Reply:
Yeah, MF21 is some leftover from 2021. Best forget about it.